content/applications/general/email_communication/email_servers.rst

```diff
@@ -129,94 +129,80 @@ How to manage inbound messages
-
-* **Bounced messages** are routed to **bounce@** in order to track them in Odoo.
-  This is especially used in `Odoo Email Marketing <https://www.odoo.com/page/email-marketing>`__ 
-  to opt-out invalid recipients.    
-
-* **Original messages**: Several business objects have their own alias to 
-  create new records in Odoo from incoming emails:
+* **Reply messages** of messages sent from Odoo are routed to their original discussion thread (and
+  to the inbox of all its followers) by the catchall alias (**catchall@**).
+* **Bounced messages** are routed to **bounce@** in order to track them in Odoo. This is especially
+  used in `Odoo Email Marketing <https://www.odoo.com/page/email-marketing>`__ to opt-out invalid
+  recipients.
+* **Original messages**: Several business objects have their own alias to create new records in
+  Odoo from incoming emails:
 
   * Sales Channel (to create Leads or Opportunities in `Odoo CRM <https://www.odoo.com/page/crm>`__),
-  
   * Support Channel (to create Tickets in `Odoo Helpdesk <https://www.odoo.com/page/helpdesk>`__),
-
-  * Projects (to create new Tasks in `Odoo Project <https://www.odoo.com/page/project-management>`__),
-
-  * Job Positions (to create Applicants in `Odoo Recruitment <https://www.odoo.com/page/recruitment>`__),
-
+  * Projects (to create new Tasks in `Odoo Project <https://www.odoo.com/page
+    /project-management>`__),
+  * Job Positions (to create Applicants in `Odoo Recruitment <https://www.odoo.com/page
+    /recruitment>`__),
   * etc.
 
-Depending on your mail server, there might be several methods to fetch emails.
-The easiest and most recommended method is to manage one email address per Odoo
-alias in your mail server.
+Depending on your mail server, there might be several methods to fetch emails. The easiest and most
+recommended method is to manage one email address per Odoo alias in your mail server.
 
-* Create the corresponding email addresses in your mail server 
+* Create the corresponding email addresses in your mail server
   (catchall@, bounce@, sales@, etc.).
 * Set your domain name in the General Settings.
 
   .. image:: email_servers/alias-domain.png
       :align: center
+      :alt: Setting the domain alias configuration on Odoo.
 
-* If you use Odoo on-premise, create an *Incoming Mail Server* in Odoo for each alias. 
-  You can do it from the General Settings as well. Fill out the form according 
-  to your email provider’s settings. 
-  Leave the *Actions to Perform on Incoming Mails* blank. Once all the 
-  information has been filled out, click on *TEST & CONFIRM*.
+* If you use Odoo on-premise, create an :guilabel:`Incoming Mail Server` in Odoo for each alias.
+  You can do it from the General Settings as well. Fill out the form according to your email
+  provider's settings. Leave the :guilabel:`Actions to Perform on Incoming Mails` blank. Once all
+  the information has been filled out, click on :guilabel:`TEST & CONFIRM`.
```

Comment:
- Replace `If you use Odoo on-premise` with `If the database's hosting type is Odoo on-premise`
- Replace `You can do it` with `This can be done`
- Replace `your email provider's` with `the email provider's`